ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.create-run.tsx

```diff
@@ -83,6 +83,10 @@ const formSchema = z.object({
     }),
     evaluator: z.object({
       enabled: z.boolean(),
+      copyrightGarbageFile: z.string().optional(),
```

Comment:
Nit: For consistency, I'd prefer if the order of the form fields was also reflected in this hunk, and in the two hunks below.